ursa/README.md

```diff
@@ -0,0 +1,6 @@
+# Ursa
+
+Ursa is a research project based on ClickHouse to make ClickHouse faster than Umbra.
```

Comment:
So far goal is not yet achieved. But this is the first try.

I tried to calculate result performance with all other possible optimizations that I plan to add. I will still be a little bit slower than Umbra, so need to research more.